### PR TITLE
Fixed compatibility issues with timer-stopwatch

### PR DESCRIPTION
**Pull request by Jackymancs4**
Actually I'm maintaining my own version of the [package](https://www.npmjs.com/package/timer-stopwatch-dev) where I fixed/merged some patches for the known problems.

I would be great if you can release a new build, since the last one doesn't support retina screens at all (due to an outdated version of the spinner, I suppose).

### CODE REVIEW

**Comment by G07cha** [comment]
Hi @Jackymancs4, thanks for your contribution! I'll go through the PR, later on, today and will merge it if everything will be fine.

**Comment by Jackymancs4** [comment]
> Looks good although timer.state as a boolean property is confusing to me.

Hello @G07cha. Yes, you're right, it was an heritage of the ```runTimer``` variable. Now it should be way more clear.

File: package.json [hunk 1]
```diff
@@ -23,11 +23,12 @@
     "menubar": "^5.2.0",
     "photon": "git+https://github.com/connors/photon.git",
     "retinajs": "^2.1.0",
-    "timer-stopwatch": "^0.2.0"
+    "timer-stopwatch-dev": "^1.0.0",
```
[the remark this answers is not in the document]
Comment (Jackymancs4) [reply package.json]: My bad, had to disable the linter and I missed it! Now is fixed.

**Comment by G07cha** [comment]
Looks good but I don't get why you have removed `package-lock.json` file?

**Comment by Jackymancs4** [comment]
Mmm.. it was a mistake actually. Sorry. I'm used to use yarn as an alternative package manager and I deleted ```package-lock.json``` out of habit to keep just one lock file. Do you want me to commit it or you'll add it later?
It's not a problem for me.

**Comment by G07cha** [comment]
Thanks for the explanation, I'll add it later.

**Comment by G07cha** [comment]
And thanks again for your contribution!

**Pull request merged**